ops/framework.py

```diff
@@ -595,7 +595,7 @@ class Framework(Object):
     """The charm's metadata."""
 
     charm_dir: pathlib.Path = None  # type: ignore
-    """The charm project root directory."""
+    """The directory where the charm is running."""
```

Comment:
I guess that's the best we can do.

I was thinking "directory where Juju deployed charm to on the current unit", but that too can be misunderstood, like does that include trailing `/charm/` or not?